apps/rush-lib/src/utilities/VersionControl.ts

```diff
@@ -2,39 +2,46 @@
 // See LICENSE in the project root for license information.
 
 import * as child_process from 'child_process';
+import * as colors from 'colors';
+import { Executable } from '@microsoft/node-core-library';
+
+const DEFAULT_BRANCH: string = 'master';
+const DEFAULT_REMOTE: string = 'origin';
+const DEFAULT_FULLY_QUALIFIED_BRANCH: string = `${DEFAULT_REMOTE}/${DEFAULT_BRANCH}`;
 
 export class VersionControl {
-  public static getChangedFolders(targetBranch?: string): Array<string | undefined> | undefined {
-    const branchName: string = targetBranch ? targetBranch : 'origin/master';
-    const output: string | undefined = child_process.execSync(`git diff ${branchName}... --dirstat=files,0`)
-      .toString();
-    return output.split('\n').map(s => {
-      if (s) {
-        const delimiterIndex: number = s.indexOf('%');
-        if (delimiterIndex > 0 && delimiterIndex + 1 < s.length) {
-          return s.substring(delimiterIndex + 1).trim();
+  public static getChangedFolders(targetBranch: string): Array<string | undefined> | undefined {
+    VersionControl._fetchNonDefaultBranch(targetBranch);
+    const output: string = child_process.execSync(`git diff ${targetBranch}... --dirstat=files,0`).toString();
+    return output.split('\n').map((line) => {
+      if (line) {
+        const delimiterIndex: number = line.indexOf('%');
+        if (delimiterIndex > 0 && delimiterIndex + 1 < line.length) {
+          return line.substring(delimiterIndex + 1).trim();
         }
       }
+
       return undefined;
     });
   }
 
-  public static getChangedFiles(prefix?: string, targetBranch?: string): string[] {
-    const branchName: string = targetBranch ? targetBranch : 'origin/master';
-    const output: string = child_process
-      .execSync(`git diff ${branchName}... --name-only --no-renames --diff-filter=A`)
-      .toString();
+  public static getChangedFiles(targetBranch: string, prefix?: string): string[] {
```

Comment:
Could we document this a little better?  What does `prefix` do? What happens if it is omitted?  What does this function return (relative or absolute paths)?